test/space_calculation_test.rb

```diff
@@ -112,6 +112,8 @@ def expect_to_execute(command)
     end
   end
 
+# storage-ng
+=begin
```

Comment:
I will add a comment to https://github.com/yast/yast-storage-ng/blob/master/doc/installer-hacks.md explaining the rational in the next update (due today).